various-receipes/message-queue/docker/docker-compose.yml

```diff
@@ -0,0 +1,9 @@
+pubsub-emulator:
+  image: 'singularities/pubsub-emulator'
+  container_name: 'pubsub-emulator'
+  environment:
+    - PUBSUB_PROJECT_ID=mq-tests
```

Comment:
Question - Does it really need a true gcp project?

Reply:
I don't think we need it here, but if we want the google's module to use the emulator instead of the real one, we need to declare those variables (both project id and emulator host)